src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/configuration/RabbitMQConfiguration.kt

```diff
@@ -36,5 +37,12 @@ data class ConnectionManagerConfiguration(
     var minConnectionRecoveryTimeout: Int = 10000,
     var maxConnectionRecoveryTimeout: Int = 60000,
     val prefetchCount: Int = 10,
-    val messageRecursionLimit: Int = 100
-) : Configuration()
\ No newline at end of file
+    val messageRecursionLimit: Int = 100,
+    val workingThreads: Int = 1,
+    val confirmationTimeout: Duration = Duration.ofMinutes(15)
```

Comment:
```suggestion
    val confirmationTimeout: Duration = Duration.ofMinutes(5)
```

Reply:
Most of our boxes should handle batches as faster as possible because they are part of conveyor